pwa/src/context/queryLimit.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import { GlobalContext } from "./global";
+
+export const queryLimitDefault = 12;
+
+export interface IQueryLimitContext {
+  objectsQueryLimit: number;

```

Comment:
Maybe this could have a more descriptive name as to _what_ kind of object (query) we're limiting here. I think this was copied from the gateway-ui, where we actually are limiting objects (which is also the name of the hook and such). 

I think `openWooObjects` or something might be better.